From: "Perry.Yuan" <perry_yuan@dell.com>
Date: Wed, 28 Apr 2021 18:37:39 +0800
Subject: hwdb: 60-keyboard:: Update Dell Privacy Local Mic Mute Hotkey
 (#19261)

This patch fix scancode 0x120001 mapping to key code F20 micmute
The previous scancode is not correct, it will cause the micmute
hotkey no function when testing the mic mute
---
 hwdb.d/60-keyboard.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 5396ddd..cece8e6 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -386,7 +386,7 @@ evdev:name:Dell WMI hotkeys:dmi:bvn*:bvr*:bd*:svnDell*:pnXPS*:*
 
 # Dell Latitude privacy microphone mute
 evdev:name:Dell Privacy Driver:dmi:bvn*:bvr*:bd*:svnDell*:pnLatitude*:*
- KEYBOARD_KEY_12001=f20                                 # Mic mute toggle, should be micmute
+ KEYBOARD_KEY_120001=f20                                # Mic mute toggle, should be micmute
 
 ###########################################################
 # Everex
